cmd/terramate/e2etests/run_signal_test.go

```diff
@@ -33,8 +33,7 @@ func TestRunSendsSigkillIfCmdIgnoresInterruptionSignals(t *testing.T) {
 	git.Add(".")
 	git.CommitAll("first commit")
 
-	tm := newCLI(t, s.RootDir())
-	tm.loglevel = "trace"
+	tm := newCLIWithLogLevel(t, s.RootDir(), "trace")
```

Comment:
We are going to keep trace level on this test ? (it is usually disabled by default)

Reply:
It was already "trace", I just used the constructor instead of setting the `tm.logLevel` by hand.

Reply:
but I think they could be disabled

Reply:
ah yeah I probably forgot then as trace =P